test/smoketest/main.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+import requests
+from server_test import *
+
+baseurl = "http://localhost:8888"
+
+def upload_cvr(filename):
+    with open(filename, 'rb') as f:
+        payload = {'county': 'Arapahoe', 'hash': 'xyzzy'}
+        r = requests.post(baseurl + "/upload-cvr-export", files={'cvr_file': f}, data=payload)
+    print(r)
+
+def upload_manifest(filename):
+    with open(filename, 'rb') as f:
+        payload = {'county': 'Arapahoe', 'hash': 'xyzzy'}
+        r = requests.post(baseurl + "/upload-ballot-manifest", files={'bmi_file': f}, data=payload)
+    print(r)
+
+def server_sequence():
+    test_get_cvr()
+    test_get_ballot_manifest()
+    test_get_contest()
+    test_get_cvr_county_Arapahoe()
+    test_get_ballot_manifest_county_Arapahoe()
+    test_get_contest_id_2()
+    test_get_contest_county_Arapahoe()
+
+if __name__ == "__main__":
+    upload_cvr("../e-1/arapahoe-regent-3-clear-CVR_Export.csv")
+    upload_manifest("../e-1/arapahoe-manifest.csv")
+    # server_sequence()
```

Comment:
Yes, for now. If I like zerotest and pytest, I assume I can figure out how to run the tests from main and update the README.md.

I've also resolved the other issues.
Both Python 2 and Python 3 work, but I'm mainly testing with Python 3.
I've made note of that.